net/wget2/Portfile

```
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wget2
+version             2.1.0
+revision            0
+
+homepage            https://gitlab.com/gnuwget/wget2
+
+description         \
+    GNU Wget2 is the successor of GNU Wget, a file and recursive website \
+    downloader.
+
+long_description    {*}${description}.
+
+categories          net www
+installs_libs       no
+license             GPL-3
```

Per https://gitlab.com/gnuwget/wget2/-/blob/master/README.md?ref_type=heads#license:

> This program is free software; you can redistribute it and/or modify
> it under the terms of the GNU General Public License as published by
> the Free Software Foundation; either version 3 of the License, or
> (at your option) any later version.

```suggestion
license             GPL-3}
```